tests/bindings/test_lqubit_bindings_nb.py

```diff
@@ -0,0 +1,338 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for LightningQubit-specific Nanobind bindings."""
+
+import numpy as np
+import pytest
+from conftest import backend
+
+# Skip all tests if not using lightning.qubit
+if backend != "qubit":
```

Comment:
shouldn't it be "lightning.qubit"? If not, when did we change this to "qubit"?! :smile: